### PR TITLE
External sources link, master

### PR DESCRIPTION
**Pull request by mvidner**
Merging #430

### CODE REVIEW

File: src/lib/y2packager/clients/inst_productsources.rb [hunk 1]
```diff
@@ -444,27 +444,23 @@ def InitializeSources
     end
 
     def ReadControlFile
+      # Prefer control.xml to /etc/YaST2/ProductFeatures (bsc#1132613)
+      ProductControl.Init
```
Comment (lslezak) [inline src/lib/y2packager/clients/inst_productsources.rb]: This client is also called [during installation](https://github.com/yast/skelcd-control-openSUSE/blob/master/control/control.openSUSE.xml#L919). Is it safe to call [ProductControl.Init](https://github.com/yast/yast-yast2/blob/master/library/control/src/modules/ProductControl.rb#L1175) in that case?

Comment (mvidner) [reply src/lib/y2packager/clients/inst_productsources.rb]: Uh, I don't know :-(

Comment (mvidner) [reply src/lib/y2packager/clients/inst_productsources.rb]: I've checked openQA how #430 is doing. `productsources` is only enabled in openSUSE, so the oenQA group to check is https://openqa.opensuse.org/group_overview/50 "Last Builds for openSUSE Leap 15". It works fine.
The reinitialization could break add-ons, but they are only brought in later.

Comment (mvidner) [reply src/lib/y2packager/clients/inst_productsources.rb]: As for logging, ProductControl.Init does call `Builtins.y2milestone("Reading control file: #{@current_control_file}")`

**Pull request merged**